### PR TITLE
Fork and rebrand cluster-api for openshift

### PR DESCRIPTION
**Pull request by flaper87**
/cc @tomassedovic @russellb @enxebre

### CODE REVIEW

File: pkg/apis/openstackproviderconfig/v1alpha1/register.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 	"sigs.k8s.io/yaml"
 
 	clusterv1 "github.com/openshift/cluster-api/pkg/apis/cluster/v1alpha1"
```
Comment (enxebre) [inline pkg/apis/openstackproviderconfig/v1alpha1/register.go]: left over?

Comment (flaper87) [reply pkg/apis/openstackproviderconfig/v1alpha1/register.go]: nope, this is used for the `ClusterSpecFromProviderSpec`

**Pull request merged**